examples/stargate-prometheus-grafana/start_stargate-prometheus-grafana.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+COMPOSE_FILE=stargate-prometheus-grafana.yml
+
+# Make sure backend-1, the seed node, is up before bringing up other nodes and stargate
+docker-compose --file $COMPOSE_FILE up -d backend-1
+sleep 60
```

Comment:
Something I recently discovered 

```sh
(docker-compose logs -f backend-1 &) | grep -q "DSE startup complete."
```

That will cause the script to wait while tailing the logs until the "DSE startup complete" message is logged.

Reply:
I changed this, but used a C* log message `now part of the of the cluster`.